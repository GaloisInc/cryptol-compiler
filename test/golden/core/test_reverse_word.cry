main = reverse 0x1234