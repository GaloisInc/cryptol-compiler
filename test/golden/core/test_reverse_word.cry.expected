0x2c48
